.github/workflows/CD.yml

```diff
@@ -19,7 +19,7 @@ on:
   push:
```

Comment:
Should a version of builds run after merge to postBuild branch?

Reply:
That would mean we're pushing to npm which we don't need/want to do.